website/templates/emails/mentions.txt.mako

```diff
@@ -0,0 +1,3 @@
+${user} at ${localized_timestamp}: ${content}.
```

Comment:
It is the same as [comment email template](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/templates/emails/comments.txt.mako). What is the notifications HTML email template? [notify_base.mako](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/templates/emails/notify_base.mako)?